IowaPD_people.yaml

```diff
@@ -1,3 +1,5 @@
+# select * from dispatch_person_all;
+
 entityDefinitions:
   Officers:
     name: "Officers"
```

Comment:
For ALL the people, can we: 
- standardize Sex, Race and Ethnicity according to the wiki
- investigate regexes
- standardize eye and haircolor in persondetails AND move that to a physicalappearance entity set instead of being in persondetails. Then, I'm also open to just not using persondetails because the only thing left is juvenile flag, if there's somewhere else to put that.

 Also let's remember to shorten all entity  set names consistent with the other flights.